commands.js

```diff
@@ -2476,8 +2476,11 @@ exports.commands = {
 		if (!this.can('hotpatch')) return false;
 		if (Monitor.hotpatchLock) return this.errorReply("Hotpatch is currently been disabled. (" + Monitor.hotpatchLock + ")");
 
-		let staff = Rooms('staff');
-		if (staff) staff.add("(" + user.name + " used /hotpatch " + target + ")").update();
+		let announceRooms = ['development', 'staff', 'upperstaff'];
+		for (let i in announceRooms) {
```

Comment:
Hm, I just double checked and I didn't see anything that "explicitly" says not to do that.

It does, however say:
>Array#forEach: NEVER - Worse performance than for-of on Arrays. See for-of.

Perhaps I should make a pull request on CONTRIBUTING.md to clarify to not use ``for..in`` on arrays?